### ISSUE TITLE
Publish on npm

### ISSUE THREAD

**Issue by DomiR** [sandeepmistry/node-bluetooth-hci-socket#85]
Will you publish the current master any time soon?

**Comment by morganrallen** [comment]
Yeah @sandeepmistry it would be great to see 0.5.2 published. I cannot get an earlier version to work on CHIP.

**Comment by rzr** [comment]
Yes an intermediary version before node10 support will not hurt

**Comment by rzr** [comment]
As explained at:
https://github.com/noble/node-bluetooth-hci-socket/issues/84

Until upstream take back the maintenance

I am proposing to co maintain a fork and eventually publish in NPM in a subrepo (org) ?
Can you suggest any ?

you can check status of the fork at:
https://travis-ci.org/abandonware/node-bluetooth-hci-socket/

**Comment by rzr** [comment]
Last call before I try to upload a revision on npm org subrepo.

Please test:
https://github.com/abandonware/node-bluetooth-hci-socket/

I am listening for any recommendations from upstream and others.

**Comment by Apollon77** [comment]
Go from my side. I only had a very short look onto it ...

**Comment by rzr** [comment]
@Apollon77  for CI check:
https://ci.appveyor.com/project/rzr/node-bluetooth-hci-socket

**Comment by Apollon77** [comment]
cool, so you should remove nodejs 4 from testing :-)

**Comment by rzr** [comment]
PR welcome ;)

**Comment by rzr** [comment]
Please review:  
https://github.com/abandonware/noble/pulls
